### DISCUSSION TITLE
Bold table header title when sorted

### DISCUSSION THREAD

**Discussion by cdecinkoKnight** [quasarframework/quasar#15561] [General - Components / Directives / etc]
When a table header is clicked to sort a column, an arrow appears but it's very subtle that the column is now sorted. We'd like to make the title bold so it's more obvious. How can we accomplish this?

**Comment by MilosPaunovic** [comment]
You can either utilize `headerStyle` and/or `headerClasses` properties from the column definition for table or you can use simple CSS to handle it the way you want it.
